src/Microsoft.Health.Fhir.Core/Features/Search/SearchValues/NumberSearchValue.cs

```diff
@@ -17,12 +17,23 @@ public class NumberSearchValue : ISearchValue
         /// Initializes a new instance of the <see cref="NumberSearchValue"/> class.
         /// </summary>
         /// <param name="number">The number value.</param>
-        public NumberSearchValue(decimal number) => Number = number;
+        /// <param name="low">The lower bound of the quantity range.</param>
```

Comment:
we could still have `NumberSearchValue(decimal number)` for convenience when there is only one value? It will just set both low and high the same number.

Reply:
Also, the `number` comment above doesn't apply anymore